src/app.scss

```diff
@@ -1,3 +1,7 @@
-@import './components/TutorialHeader/tutorial-header.scss';
-@import './content/LandingPage/landing-page.scss';
-@import './content/RepoPage/repo-page.scss';
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/type/scss/font-family.scss';
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/layout/scss/breakpoint.scss';
+@import 'carbon-components/scss/globals/scss/typography.scss';
+@import 'carbon-components/scss/globals/scss/vars.scss';
+@import './components/TutorialHeader/_tutorial-header.scss';
+@import './content/LandingPage/_landing-page.scss';
+@import './content/RepoPage/_repo-page.scss';
```

Comment:
You should be importing the versions without the underscores. Please try again and make sure your layout matches the preview from step 2 before committing your updates to this PR.